### ISSUE TITLE
Skin Tone Classifier Execution in Google Colab"

### ISSUE THREAD

**Issue by Arslankmb** [ChenglongMa/SkinToneClassifier#6]
Issue Description:

I encountered an issue when using the Skin Tone Classifier library in a Google Colab environment. The problem appears to be related to Qt platform plugins, and I received the following error messages:
**qt.qpa.plugin: Could not find the Qt platform plugin "offscreen" in "/usr/local/lib/python3.10/dist-packages/cv2/qt/plugins"
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
Available platform plugins are: xcb.**

**Steps to Reproduce**:
Install the Skin Tone Classifier library using pip install skin-tone-classifier --upgrade.
Run the following command in a Jupyter Notebook or Google Colab environment:
**stone -i /path/to/image.jpg --debug**

**Expected Behavior:**
I expected the Skin Tone Classifier to process the image and provide results as described in the library's documentation.

**Actual Behavior:**
Instead, I encountered the error messages mentioned above, which prevented the library from functioning correctly.

**System Information:**
Operating System: Google Colab (Colab's default environment)
Python Version: 3.10
Skin Tone Classifier Version: 1.0.0

**My attempts**
I attempted to resolve this issue by following the suggestions in the error message, but it did not resolve the problem. The issue seems to be related to the Qt platform plugins, specifically "offscreen."

Please let me know if there are any specific logs or additional information needed to diagnose and address this problem.
Thank you for your assistance.

**Comment by ChenglongMa** [comment]
Hi @Arslankmb, thanks for your feedback!

## TL;DR

1. You can ignore this error because it won't affect the results. You can still find the result files in the `debug` folder and `result.csv`.
2. When you process **multiple** images, no error will be reported. The error is only popped up when processing **one** image.
3. I will fix this issue in the next version.

Thanks!

-----
## Detailed explanation

`stone` was originally designed for command line use, not for Jupyter Notebook, so I hadn't considered this issue before.

I have tested the library in a Google Colab notebook and included some working examples (You can also check it by clicking [here](https://colab.research.google.com/drive/1k-cryEZ9PInJRXWIi17ib66ufYV2Ikwe?usp=sharing)). 
I found that this issue is caused by the `OpenCV` being incompatible with web browsers (like Google Colab, and Jupyter Notebook). 

This issue occurs only when processing a **single** image, as `stone` specifically displays the resulting window when only **ONE** image is being processed.

I will fix this issue in the next version.

Thanks for your attention and patience!

**Comment by ChenglongMa** [comment]
Hi @Arslankmb,

Finally, I decided to remove this small function. Please run `pip install skin-tone-classifier --upgrade` to upgrade the library to v1.0.1. It won't show the resulting window when processing a single image and also won't report the error in notebooks.

Thanks!

**Comment by ChenglongMa** [comment]
For those looking for help and find this:
`stone` now can be imported into other projects or Jupyter Notebook env as a library since version 1.1.0.

Please run the following command to upgrade it to version 1.1.0.
```bash
pip install skin-tone-classifier --upgrade
```

Following [README#9-used-as-a-library-by-importing-into-other-projects](https://github.com/ChenglongMa/SkinToneClassifier#9-used-as-a-library-by-importing-into-other-projects) to import `stone`, e.g., 

```python
import stone

result = stone.process('example.jpeg', , image_type='color')
```

Then the `result` would be:

```json
{
  "basename": "example",
  "extension": ".jpeg",
  "image_type": "color",
  "faces": [
    {
      "dominant_colors": [
        {
          "color": "#D7B19F",
          "percent": "0.62"
        },
        {
          "color": "#7A4E3B",
          "percent": "0.38"
        }
      ],
      "skin_tone": "#BEA07E",
      "tone_label": "CG",
      "accuracy": 82.27,
      "face_id": 1
    }
  ],
  "report_images": {
    "1": null
  }
}
```

Thanks!
